docs/case-studies/gitops.md

```diff
@@ -24,26 +24,101 @@ In this section, we will introduce steps of using KubeVela directly in GitOps ap
 
 > Note: you can also use it with existing tools such as ArgoCD with similar steps, detailed guides will be added in following releases.
 
-## Setup
+## Setup Code Repository
 
-First, setup a Git Repository with `Application` files, some source code and a Dockerfile.
+First, setup a Git code repository with some source code and a Dockerfile.
 
-The code is very simple, starting a service and displaying the version in the code. In `Application`, we'll start a `webservice` for the code and add an `Ingress` trait to access.
+The code is very simple, starting a service and connect to a MySQL database. In the default path, it will display the version in the code; in the `/db` path, it will list the data in database.
```

Comment:
```suggestion
The deployment looks like this: it starts a service and connects it to a MySQL database. In the default path, it will display the version in the code; in the `/db` path, it will list the data in database.
```
```suggestion
The app serves HTTP service and connects to a MySQL database. In the '/' path, it will display the version in the code; in the `/db` path, it will list the data in database.
```